packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details/paths.dart

```diff
@@ -8,15 +8,15 @@ import '../../../../../../primitives/utils.dart';
 import '../../../../../../shared/table/table.dart';
 import '../../../../../../shared/table/table_data.dart';
 import '../../../../../../shared/utils.dart';
+import '../../../../primitives/simple_elements.dart';
 import '../../../../shared/heap/heap.dart';
-import '../../../../shared/heap/primitives.dart';
 
 class _RetainingPathColumn extends ColumnData<StatsByPathEntry> {
   _RetainingPathColumn(String className)
       : super.wide(
           'Shortest Retaining Path for Instances of $className',
-          titleTooltip: 'Class names of objects that retain'
-              '\nthe instances from garbage collection.',
+          titleTooltip: 'The shortest of sequences of objects that retain'
```

Comment:
May be the phrasing should be improved other way though.

Reply:
I would say "The shortest sequence" instead of "The shortest of sequences" as these are equivalent and the former is easier to understand and more terse.